src/app/index.html

```diff
@@ -1,3 +1,18 @@
+<h2>
+  Docs
+</h2>
+<ul>
+  <li>
+    <a skyAppLink="/docs/passive-indicator">Passive indicator</a>
```

Comment:
Doesn't really matter that much, but I'd add `progress` in the middle here. And same for the next item.